Packs/MalwareInvestigationAndResponse/Scripts/KillProcessWrapper/KillProcessWrapper.yml

```diff
@@ -0,0 +1,46 @@
+args:
+- default: false
+  description: ' The Endpoint ID in which you would like to kill the given process.'
+  isArray: false
+  name: endpoint_id
+  required: true
+  secret: false
+- default: false
+  description: 'The ID of the process to kill. '
```

Comment:
I think we should make it clear that either the id or the name must be provided.

Reply:
Added that.